jupyter_docx_bundler/converters.py

```diff
@@ -170,7 +170,8 @@ def preprocess(content, path, handler=None):
         # process outputs
         if 'outputs' in cell and any(['data' in output for output in cell['outputs']]):
```

Comment:
Please remove the additional check for data in this if-clause. We do this better in your code below.